### ISSUE TITLE
which part in code is cross-attention mask and temporal attention mask?

### ISSUE THREAD

**Issue by Nutingnon** [microsoft/Peekaboo#6]
Hi, I'm trying to learn your implementation for your amazing paper and I have some questions:

I see there are two mask, one is `frozen_mask` , another is `encoder_attention_mask`
I guess the frozen_mask is the one for spatial-temporal, which is the foreground boundingbox
the encoder_attention_mask is for text condition.

However, the question is:
- 1. [Masked spatial attention]  I think the frozen_mask can only mask the foreground (bbox) pixels, not the background pixels, because I didnot find the code for masking background one
- 2. [masked cross-attention]   I cannot find the code that frozen_mask interacts with encoder_attention_mask, which is masked cross-attention
- 3. [masked temporal attention] I cannot find the code for masked temporal attention. 

Thank you for anyone who can help me
